modules/apps/commerce/commerce-warehouse-web/src/main/java/com/liferay/commerce/warehouse/web/internal/display/context/CommerceInventoryWarehousesDisplayContext.java

```diff
@@ -135,25 +116,41 @@ public CommerceInventoryWarehouse getCommerceInventoryWarehouse()
 		return _commerceInventoryWarehouse;
 	}
 
+	public Country getCountry(long countryId) throws PortalException {
+		return _countryService.getCountry(countryId);
+	}
+
+	public Country getCountry(String countryTwoLettersIsoCode)
+		throws PortalException {
+
+		return _countryService.getCountryByA2(
+			_cpRequestHelper.getCompanyId(), countryTwoLettersIsoCode);
+	}
+
+	public String getCountryTwoLettersIsoCode() {
```

Comment:
It seems inconsistent between when we choose to use portal Country's field names and when we use the legacy CommerceCountry field names. This could be changed to `getCountryA2`.